### PR TITLE
getMe() Telegram method for Bot class was implemented

### PR DESCRIPTION
**Pull request by shadrus**

### CODE REVIEW

File: aiotg/bot.py [hunk 1]
```diff
@@ -220,6 +220,15 @@ def group(self, group_id):
...
 
+    _get_me = partialmethod(api_call, "getMe")
+
+    def get_me(self):
+        """
+        Returns basic information about the bot
+        (see https://core.telegram.org/bots/api#getme)
+        """
+        return self._get_me()
+
```
Comment (szastupov) [inline aiotg/bot.py]: I guess partialmethod is not necessary here, could you remove _get_me and instead, use api_call directly from get_me()?

**Comment by szastupov** [comment]
Oh, and next time, could you rebase you branches on top of master so that changes will be isolated from each other, because right now I see changes related to other PRs and it's getting harder to review.

Anyway, thanks a lot for all the contributions!

**Comment by szastupov** [comment]
Thanks!

**Comment by shadrus** [comment]
You are welcome!

**Pull request merged**